tests/lib/rules/no-array-prototype-extensions.js

```diff
@@ -565,10 +565,120 @@ import { get as g } from 'dummy';
       errors: [{ messageId: 'main', type: 'CallExpression' }],
     },
     {
+      // When unexpected number of arguments are passed, auto-fixer will not run
       code: 'something.sortBy()',
       output: null,
       errors: [{ messageId: 'main', type: 'CallExpression' }],
     },
+    {
+      // Arguments containing strings
+      code: "something.sortBy('abc', 'def')",
+      output: `import { compare } from '@ember/utils';
```

Comment:
Can we make this a lot more concise? I don't think we need one-time-use variables, comments, or newlines.

Here's what I would output:

```js
something.sort((a, b) => {
  for (const key of ['abc', 'def']) {
    const compareValue = compare(get(a, key), get(b, key));
    if (compareValue) {
      return compareValue;
    }
  }
  return 0;
});
```

Reply:
Makes sense. I just copied over the existing implementation of the `sortBy` function. Will make it concise as you suggested.